vite.config.js

```diff
@@ -37,13 +37,13 @@ export default defineConfig({
     sourcemap: 'inline',
   },
   test: {
-    environment: 'jsdom',
+    environment: 'dom',
     globals: true,
-    setupFiles: 'src/setupTests.ts',
+    setupFiles: 'src/setupTestNew.ts',
   },
   resolve: {
     alias: [
-      { find: '@/', replacement: '/src/' },
+      { find: '@/', replacement: '/srce/' },
```

Comment:
**Suggestion:** Correct the path replacement in the alias. It should be '/src/' instead of '/srce/'.
```suggestion
      { find: '@/', replacement: '/src/' }
```